packages/modul-components/src/components/dropdown/dropdown.ts

```diff
@@ -405,7 +407,11 @@ export class MDropdown extends BaseDropdown implements MDropdownInterface {
 
     private async selectText(): Promise<void> {
         await this.$nextTick();
-        this.$refs.input.focus();
+        if (!this.isAndroid && this.as<MediaQueriesMixin>().isMqMaxS) {
```

Comment:
idéalement éliminer la duplication du code vers vers une méthode is[nom_significatif]

Reply:
pas nécessaire je check que sur isAndroid maintenant.